model/inline/src/main/scala/aqua/model/inline/ArrowInliner.scala

```diff
@@ -39,7 +39,7 @@ object ArrowInliner extends Logging {
   private def getOutsideStreamNames[S: Exports]: State[S, Set[String]] =
     Exports[S].exports
       .map(exports =>
-        exports.collect { case (n, VarModel(_, StreamType(_), _)) =>
+        exports.collect { case (n, VarModel(_, _: MutableStreamType, _)) =>
```

Comment:
On the second thought, I don't understand, is it needed here?
Result of this function is used in `pushStreamResults` which works only with `StreamType`.
Moreover, semantic would reject code with map on the left side of function call:
```aqua
aqua Main

export main

func getString() -> string:
  <- "testing"

func main():
  myMap: %string
  myMap <- getString()
```
gives error:
```
10   myMap <- getString()
     ^^^^^===============
        This name was already defined in the scope
```